src/screens/TopicMemberScreen/index.js

```diff
@@ -1,5 +1,12 @@
 import * as React from 'react';
-import {Animated, InteractionManager, ScrollView, StatusBar, StyleSheet} from 'react-native';
+import {
+  Animated,
+  InteractionManager,
+  Keyboard,
+  ScrollView,
+  StatusBar,
+  StyleSheet
+} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been expanded to multiple lines. This change doesn't affect the functionality of the code, but it does make it more readable by making it easier to see at a glance what modules are being imported from 'react-native'.
```diff
- import {Animated, InteractionManager, ScrollView, StatusBar, StyleSheet} from 'react-native';
+ import {
+   Animated,
+   InteractionManager,
+   Keyboard,
+   ScrollView,
+   StatusBar,
+   StyleSheet
+ } from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->